src/main/java/com/backbase/ct/bbfuel/util/ResponseUtils.java

```diff
@@ -28,6 +31,14 @@ public static boolean isBadRequestException(Response response, String withMessag
         return response.statusCode() == SC_BAD_REQUEST && getBadRequestMessage(response).equals(withMessage);
     }
 
+    public static boolean isConflictException(Response response, String withMessage) {
+        return response.statusCode() == SC_CONFLICT && getBadRequestMessage(response).equals(withMessage);
+    }
+
+    public static boolean isNotFoundException(Response response, String withMessage){
+        return response.statusCode() == SC_NOT_FOUND && getBadRequestMessage(response).equals(withMessage);
```

Comment:
this will cast the Not Found exception as a BadRequestException. They might be compatible but still feels wrong.